### PR TITLE
[AS-4497] Adding tracking to notification page

### PR DESCRIPTION
**Pull request by daytavares**
The type of this PR is: **Feature**

This PR resolves [[EMI-1783](https://artsyproduct.atlassian.net/browse/EMI-1783)]

### PR Checklist (tick all before merging)

- [ ] If I've added a new file to the tree I've exported it from the common [`index.ts`](https://github.com/artsy/cohesion/blob/main/src/Schema/Events/index.ts)
- [ ] I've added comments with examples for any new interfaces and ensured that they're in the docs
- [ ] No platform-specific terminology has been used outside of `click` and `tap` (platform is inferred by the DB storing events)

[EMI-1783]: https://artsyproduct.atlassian.net/browse/EMI-1783?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Schema/Events/Click.ts [hunk 1]
```diff
@@ -2107,3 +2107,27 @@ export interface ClickedOnLearnMore {
...
+ *  }
+ * ```
+ */
+ export interface ClickedViewWork {
+  action: ActionType.clickedViewWork
+  context_owner_type: string
+  owner_id: string
+  artwork_id: string
+  notication_type: string
+}
```
Comment (MrSltun) [inline src/Schema/Events/Click.ts:2132]: **Suggestion:** Can we please also add this event for `Tap.ts`?
(as far as I know, `click` is for web & `tap` is for app)

Comment (daytavares) [reply src/Schema/Events/Click.ts:2132]: I added Context Module and Owner Type too

**Review by MrSltun — APPROVED** [review]
Looks good!! Thank you Dayane ✨

**Comment by artsyit** [comment]
:rocket: PR was released in [`v4.173.0`](https://github.com/artsy/cohesion/releases/tag/v4.173.0) :rocket:

File: src/Schema/Events/Tap.ts [hunk 2]
```diff
@@ -1063,3 +1063,29 @@ export interface TappedNewsSection {
...
+ *  }
+ * ```
+ */
+ export interface TappedViewWork {
+  action: ActionType.tappedViewWork
+  context_module: ContextModule
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id: string
+  artwork_id: string
+  notication_type: string
```
Comment (narikazu) [inline src/Schema/Events/Tap.ts:1090]: It looks like a typo. It should be notification_type.

File: src/Schema/Events/Tap.ts [hunk 3]
```diff
@@ -1063,3 +1063,29 @@ export interface TappedNewsSection {
...
+/**
+ * A user clicks on View Work on the notification page.
+ *
+ * This schema describes events sent to Segment from [[tappedViewWork]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedViewWork",
+ *    context_module : "Notifications"
```
Comment (narikazu) [inline src/Schema/Events/Tap.ts:1076]: question: It seems we don't have `notifications` context_module. We do have `notification`, though.

File: src/Schema/Events/Tap.ts [hunk 4]
```diff
@@ -1063,3 +1063,29 @@ export interface TappedNewsSection {
...
+ *    context_screen_owner_type: "notifications",
```
Comment (narikazu) [inline src/Schema/Events/Tap.ts:1077]: ditto.

**Pull request merged**